### ISSUE TITLE
[EUSM] Newly created service points are not available on the mobile app

### ISSUE THREAD

**Issue by AnnieMungai** [OpenSRP/opensrp-client-eusm#137]
- [ ] There were several service points that we created on web (about 2 weeks ago) but are not available on EUSM Mobile. See example below:
DRPPSPF Mahajanga 
Warehouse   
Madagascar > BOENY > Mahajanga I > Mahajanga I 329b9055-d706-42f4-8ef5-ed60c2f53378

- [x] User cannot create new service points (due to issue #138 )

**Comment by Naima-Bashir** [comment]
This issue is still unresolved. Test report done on 10th Janury 2024. QA Results can be found below:

1. Log into the web application:
2. Select the service point inventory on the side panel
3. Add a new service point e.g Test QQ
4. Select the parent. For example Madagascar- Androy- Ambanisarika
5. Select the type e.g water point and click save
6. Once the new service point is created, add a new inventory item to it. I added a vacuum extractor. Inventory is also know as a product (hence the clients email)
7. Log into the android application
8. Select the mission e.g Test QA
9. Select the region (this would be Androy in our case)
10. Sync the application
11. Once the sync is complete, go to list view
12. Search for the newly created service point; Test QQ

EXPECTED RESULTS: The newly created service point should be seen on the android application after the sync is complete. 
ACTUAL RESULTS: The newly created service point cannot be found on the android app. When one searches for it on the web app however, they are able to see it. See images below.
![image](https://github.com/opensrp/opensrp-client-eusm/assets/61143692/b91b6c52-1f29-4480-b849-0b489307bcc8)

**Comment by joyce-x-chen** [comment]
Thank you @Naima-Bashir! Additional information shared by the client with screenshots is available [here](https://drive.google.com/file/d/1HPa8F2R5jWqEW7uAz7TFwkO8Z-NALtn3/view?usp=drive_link).

**Comment by AnnieMungai** [comment]
Service points created but not available on the app (production)
<html xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/ANNEMU~1/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/ANNEMU~1/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
<style>

</style>
</head>

<body link="#1155CC" vlink="#1155CC">


Service   point | ID
-- | --
DRPPSPF Sambava | 7b20f500-2229-4bbe-9f67-be07514169e2
DRPPSPF Toliary I | 9446dd4e-7780-412c-9669-9d1f5b102846
CUA Antananarivo Renivohitra | f7602d80-1d8f-4adf-8f57-498159a50763
DRPPSPF Taolagnaro | 9d035351-80c8-457b-85fb-d0cfe245eae7
DRPPSPF Ambovombe-Androy | 50403427-020b-4fa6-9a40-3aa4ea1cd92a
DRPPSPF Bekily | c25d09ea-bef1-4588-9e06-c1cc89fe0724
DRPPSPF Morombe | d1622bd5-18f7-4e77-90b1-fc71694e9537
DRPPSPF Mananjary | 4009d6ad-665c-4469-83e5-dae38574d05a
DRPPSPF Manakara Atsimo | 890de7d2-d662-47b0-822a-92696b5bfc31
DRPPSPF Majunga | 329b9055-d706-42f4-8ef5-ed60c2f53378
DRPPSPF Farafangana | 447de2ff-4c1c-406c-a7a8-426de375e635
Container Manakara Atsimo | 48365772-08d7-465c-9637-792aceccda2c
DREAH Ambato Boeni | f6257267-5182-4101-a545-09880a0bb483
DREAH Ambovombe | faf8ba20-460e-4e7d-a2c9-65334838cc12
DREAH Ampanihy Ouest | a775b1af-18f9-4491-9452-fdd5438e6546
DREAH Antananarivo   Renivohitra | 8796845a-9fed-49cc-b21c-03cae436e1f6
DREAH Farafangana | 214e5fae-46fd-443b-92b7-2b00be6c86d0
DREAH Ihosy | 6aee73f5-6003-4a7f-a191-229545f6db01
DREAH Mahajanga | 7f775600-e586-43b6-92b3-a7aeb419a983
DREAH Manakara Atsimo | b5555254-3316-47a4-a047-50615572a2a2
DREAH Mananjary | f5a826f2-5fda-46a7-a706-e10e3eee52ea
DREAH Marovoay | 292002f0-d59a-4bf6-b4b5-f95a0998c35d
DREAH Midongy Atsimo | 682a3a69-67e7-459b-9185-b51a713219a6
DREAH Nosy Varika | f7c970a3-1301-47e5-b563-b81ef0867f87
DREAH Taolagnaro | 2b652391-d2af-435b-a482-09ce3cfe2b93
DREAH Toliara | 09b2f577-144f-4c4e-bc9a-85060759d233
SAF FJKM Manakara Atsimo | 0ae07567-4638-4b57-b588-a77b6df8c4dc
WATERAID Fenerive Est | 9611725d-c88f-4594-840c-a236e6a22be3
MEN TANA | 524e6add-bb19-45cf-a3cb-dd71dbd5db51
DREN Androy | 1018ecef-d001-481b-ac95-bed9a5157da0
CISCO Ampanihy | f9e6c749-e179-4b8e-abe2-bd37c78d241b
DREN Boeny | 4195739d-f569-48ad-a3e1-15ef381d5f57
CISCO Farafangana | 2f6a3331-8491-498b-8089-187b8c2e0f93
DREN Fenerive Est | 70999acd-cd91-4c90-85a6-daafafd5ecea
DREN Manakara | 4264b601-6cba-49d4-9740-1c03ae9542a9
CISCO Mananara | 051f2ef4-cd89-482d-b81d-e8707792ca3b
CISCO Mananjary | b497e0d0-8ce8-4f18-8f8f-27ad0ab0001c
CISCO Maroantsetra | 0ba4373c-7ef7-48b7-a2ad-135a8b57ced6
CISCO Morombe | ef07ae53-95f1-43b2-90df-35ab32f02463
CISCO Sakaraha | d9d3d81b-1607-410e-b5d6-b6b6f231ac3a
DREN Anosy | f82cf27f-fbb8-455f-a56b-552ac79cb2ff



</body>

</html>


Could be related to: https://github.com/opensrp/opensrp-client-eusm/issues/133#issuecomment-1849740618 cc @hilpitome

**Comment by AnnieMungai** [comment]
@hilpitome on stage - newly created SPs are not seen on the app, even when the SP has assigned products. 
Examples:
Green Village warehouse. Service point ID: dbf22441-7e8f-47f6-8f12-c212a42dc6fc
Happy Kids. SP ID: 3cceed59-1e26-4dd5-a22f-a38d59d4279b

**Comment by hilpitome** [comment]
@AnnieMungai you can test if this is fixed on the staging server by opensrp-server-web [v2.10.12-ALPHA1-SNAPSHOT](https://github.com/onaio/infrastructure/pull/7704)

**Comment by AnnieMungai** [comment]
@hilpitomeStage:  newly created SPs with their products are automatically showing on the app. LGTM!
![Screenshot_2024-01-25-15-18-56-148_org smartregister eusm debug](https://github.com/opensrp/opensrp-client-eusm/assets/58034047/50309065-90d1-457d-8833-c922f9de826d)

Next steps: Deploy on prod.

**Comment by Naima-Bashir** [comment]
@AnnieMungai @hilpitome  This has failed QA. Find my QA results below: 
I've created these two service points in Annie's Mission - `Test QA Service Point `and `Test Boeny `and they are not showing on the production android application. See images below:
![Screenshot_20240201_133511_eusm](https://github.com/opensrp/opensrp-client-eusm/assets/61143692/481da558-3188-47d4-862b-35e5f156cd31)

![Screenshot_20240201_133525_eusm](https://github.com/opensrp/opensrp-client-eusm/assets/61143692/17fd0170-4a05-4958-b9e2-2f2dcf8d3988)

**Comment by Naima-Bashir** [comment]
The two service points are now loading. @hilpitome do you know how long it takes for the service points to show up on the android application. 
![Screenshot_20240201_174119_eusm](https://github.com/opensrp/opensrp-client-eusm/assets/61143692/523aff6b-c596-48f3-a32e-66a24d097210)

**Comment by hilpitome** [comment]
@Naima-Bashir I think the issue is not how long sync takes but rather how it is communicated. I have noticed a bug where you get a  series of toasts saying that sync is complete yet it is still ongoing. Therefore one might check on  a service point when it has not actually been synced yet.

**Comment by AnnieMungai** [comment]
Thanks @Naima-Bashir @hilpitome I will go ahead and close this issue. I have created a separate ticket for the sync issue.
